### ISSUE TITLE
The feature of blocking advertisements aren't working anymore after updates today in KitKat.(Ver. 1.8.0)

### ISSUE THREAD

**Issue by Kara-ism** [chiehmin/MinMinGuard#22]
Hi FatMinMin,
There are some problems in updates today,which causes advertisements can't be hidden as before.
I've uploaded an image for example;It's a popular application in Taiwan called JPTT(com.joshua.jptt).
So now,I turn back to use old version 1.7.5.
Need your help to fix it,thanks.

The environment I used is LG GPRO(E988),official KitKat Rom.
Xposed version 2.6.1
MinMinGuard 1.8.0
Also,advertisements in an app called 雙鐵時刻表(idv.nightgospel.TWRailScheduleLookUp) can't be blocked correctly,and I need your help to block it.Thanks.
![screenshot_2015-02-17-09-43-44](https://cloud.githubusercontent.com/assets/9511279/6223221/d7f83b7c-b69e-11e4-8257-bb6732931d1b.png)

**Comment by chiehmin** [comment]
I don't know why. But MinMinGuard is working fine for me on Lollipop, I don't have a KitKat device for me to test currently.  Can you try to fully uninstall and reinstall MinMinGuard 1.8.0?

**Comment by Kara-ism** [comment]
Okay,I've followed the instruction you replied;
I removed MinMinGuard fully and then reinstalled it back as latest version(1.8.0);
However,It left a blank space instead.
It seems like that there is a compatibility problem about the new ads removing method on MinMinGuard 1.8.0 in KitKat Rom.
(It works fine without blank space left in old version 1.7.5)
BTW,Happy Chinese New Year! :smile: 
![screenshot_2015-02-17-14-36-33-0](https://cloud.githubusercontent.com/assets/9511279/6224367/b207fe68-b6b5-11e4-96f4-f1a93065168d.png)
![screenshot_2015-02-17-14-44-12-0](https://cloud.githubusercontent.com/assets/9511279/6224368/b20abdd8-b6b5-11e4-83ab-f4c83844ab0f.png)
